src/app/pages/storage/components/manager/manager.component.ts

```diff
@@ -540,7 +540,7 @@ export class ManagerComponent implements OnInit, AfterViewInit {
   }
 
   canSave(): boolean {
-    if (this.isNew && !this.nameControl.value) {
+    if (this.isNew && (!this.nameControl.value || this.nameControl.value.length > 50)) {
```

Comment:
should we disable the form if existing (not new) form nameControl.value.length > 50 ??

currently it is like
new form has name limit of 50
existing form does not have any limit

Reply:
On the existing form, you can't change name so the validation doesn't matter there.